src/components/implementation/no_interface/cFE_booter/osfiles.c

```diff
@@ -26,70 +33,71 @@ int32 OS_creat(const char *path, int32  access)
 */
 int32 OS_open(const char *path,  int32 access,  uint32 mode)
 {
-    if (access != OS_READ_ONLY) PANIC("Read Only Filesystem!");
-    if (!path) return OS_FS_ERR_INVALID_POINTER;
-    if (strlen(path) > OS_MAX_PATH_LEN) return OS_FS_ERR_PATH_TOO_LONG;
-    return open(path);
+    int32 ret = chk_path(path);
+    if (ret) return ret;
+    return file_open(path);
 }
 
 /*
  * Closes an open file.
 */
 int32 OS_close(int32 filedes)
 {
-    return close(filedes);
+    return file_close(filedes);
 }
 
 /*
  * Reads nbytes bytes from file into buffer
 */
 int32 OS_read(int32 filedes, void *buffer, uint32 nbytes)
 {
-    return read(filedes, buffer, nbytes);
+    return file_read(filedes, buffer, nbytes);
 }
 
 /*
  * Write nybytes bytes of buffer into the file
 */
 int32 OS_write(int32  filedes, void *buffer, uint32 nbytes)
 {
-    PANIC("Read Only Filesystem!");
-    return 0;
+    return file_write(filedes, buffer, nbytes);
```

Comment:
Note that this will be essential once the fs stuff is a non-cFE-specific library.  So I don't know if the question is if we should have wrapping, rather how to do it best